data/coalition outfits.txt

```diff
@@ -30,6 +34,8 @@ outfit "Large Collector Module"
 
 outfit "Small Reactor Module"
 	category "Power"
+	licenses
+		Coalition
```

Comment:
Yes, the reactors should be Heliarch items. Of course, they aren't sold anywhere, so it won't matter much what license they're under.